dotcom-rendering/src/web/components/HeaderSingleFrontDoor.tsx

```diff
@@ -46,14 +47,18 @@ export const HeaderSingleFrontDoor = ({
 				headerTopBarSearchCapiSwitch={headerTopBarSearchCapiSwitch}
 			/>
 		</Island>
+
 		<div
 			css={[
 				center,
 				css`
-					overflow: auto;
+					overflow: hidden;
```

Comment:
I hope this won't affect anything around the header - it seems to be needed to prevent the snowflakes from overflowing the container as they temporarily go off screen.

Is there a way to also put this behind the #Snow flag?

Reply:
Not sure why it was added and added the question in https://github.com/guardian/dotcom-rendering/pull/6719#discussion_r1045626404

Reply:
Hi 👋 The overflow needs to be set to either auto or hidden or the mobile breakpoint will not render very nicely:

With margin auto            |  Without margin auto
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/2510683/207592691-176887ba-9c9c-469f-8882-7c575f5a9903.png)  |  ![](https://user-images.githubusercontent.com/2510683/207592787-7c83ecc6-28c2-4a16-8086-1ffc3923a58a.png)

Reply:
great, thanks for taking the time to confirm this 🙂 So it looks like we can use `hidden` as a solution here as it achieves the same effect as `auto` in this context.

It seems like the change could conceivably cause problems in the future if the number/size of the items are increased so they overflow and are subsequently hidden, but this experiment won't block changing it back if needed